base/components/LoginWidget.jsx

```diff
@@ -151,21 +152,37 @@ const RegisterLink = ({className, ...props}) => <LoginLink
 />;
 
 
+const RegisteredThankYou = () => {
+	const user = Login.getUser();
+	const name = user.name || user.xid;
+	return (<>
+		<h3>Thank you- and welcome!</h3>
+		Welcome to Good-Loop, {name}!<br/>
+		Check out your new account <a href="https://my.good-loop.com/#account">here</a>.
+	</>);
+};
 
 /**
 	Log In or Register (one widget)
 	See SigninScriptlet
 	@param render {?JSX} default: LoginWidgetGuts
 	@param logo {?String} image url. If unset, guess via app.service
 */
-const LoginWidget = ({showDialog, logo, title, Guts = LoginWidgetGuts, services}) => {
+const LoginWidget = ({showDialog, logo, title, Guts = LoginWidgetGuts, services, onLogin, onRegister}) => {
 	const show = getShowLogin();
+	const [showThankyou, setThankyou] = useState(false);
+	if (!show && showThankyou) setThankyou(false);
```

Comment:
This line could use a comment -- it feels like its fixing some odd behaviour

Reply:
Done